plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -286,6 +286,7 @@ class read_only {
 
    struct get_block_params {
       string block_num_or_id;
+      bool is_backup;
```

Comment:
change to "std::optional<bool> is_backup", for being compatible with old version